apps/emqx/src/emqx_persistent_session_ds.hrl

```diff
@@ -68,6 +76,17 @@
 }).
 -type ds_pubrange() :: #ds_pubrange{}.
 
+-record(ds_marker, {
```

Comment:
It would be nice to have a comment explaining what "marker" is. Since `_MarkerName` is a free variable, it's not obvious from the definition that currently it currently has a single value of `rec` and that it tracks the last session sequence acked by the client.

Maybe even consider renaming the record to "ds_committed_offset", and "marker name" to "commit type". But it's entirely optional.